temboardui/handlers/settings/group.py

```diff
@@ -176,7 +183,7 @@ def delete_group(self, group_kind):
             raise TemboardUIError(400, "Group name field is missing.")
         delete_group(self.db_session, data['group_name'], group_kind)
         self.db_session.commit()
-        self.db_session.close()
+        self.tearDown()
```

Comment:
You may want to remove the line above since `commit` is already done in `tearDown`.